<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

 <arg name="addpose_topic" default="/initialpose"/>
 <arg name="posearray_topic" default="/waypoints"/>
 <arg name="goal_frame_id" default="odom"/>

  <node pkg="waypoints_nav" type="waypoints_nav" name="waypoints_nav" output="screen" clear_params="true">
    <param name="goal_frame_id" value="$(arg goal_frame_id)"/>
    <param name="addpose_topic" value="$(arg addpose_topic)"/>
    <param name="posearray_topic" value="$(arg posearray_topic)"/>
  </node>
</launch>

